wiki/sensing/camera-imu-calibration.md

```diff
@@ -0,0 +1,102 @@
+3. ROS drivers for the IMU and camera. Pixhawk, Ardupilot is typically used for UAVs and UM6/7 is popular for ground robots like Husky. ZED, Asus Xtion Pro, Intel Realsense D435 are few popular stereo cameras.
+
+## Calibration Procedure
+Download and extract the CDE package.
+```
+  tar xfvz kalibr.tar.gz
+```
+
+Either you can run the tools directly from the cde-package folder or/and add the package folder to the system path using: 
+```
+  export PATH="/cde/package/path:$PATH"
+```
+Change the AprilTag size in the YAML file and copy the same yaml file inside the extracted kalibr folder. <br/> Now make sure that both IMU & Camera publishes raw data and images respectively to ROS. Below command will list all the ROS topics published by the sensor.
+```
+rostopic list
+```
+Before creating ROS bag check if the sensor data is published properly.
+```
+rostopic echo /topic_name
+```
+Check the publishing rate of a topic. It is recommended that the camera should run at 20 Hz and IMU at 200 Hz.
+```
+rostopic hz /topic_name
+```
+Further, move the robot along all its degree of freedom. For instance, the axes of the UAVs are translated in x,y & z direction & rotated in all the three directions (roll,pitch & yaw) for the proper calibration. Collect the IMU sensor’s raw measurement and camera frame for around 60 seconds. Before recoding camera data, ensure that the RGB images from the camera is converted into the grayscale format. 
+
+In this sample example, Pixhawk sensor data is subscribed from the MAVROS raw_sensor message & camera frames are subscribed from the ZED sensor camera node. For other IMUs/Cameras only the ROS message name will change.
+
+Start ZED sensor camera
+```
+roslaunch zed_wrapper zed.launch
+```
+and similarly, start pixhawk
+```
+roslaunch mavros px4.launch
+```
+Below command will start the recording.
+```
+rosbag record -O camera-imu-data.bag /imu/data_raw /zed/left/image_raw /zed/right/image_raw
+```
+Check the recorded data by using the following command.
+```
+rosbag play <your bagfile>
+```
+If all the required data is recorded properly, run following camera calibration command.
+```
+./kalibr_calibrate_cameras --models pinhole-equi pinhole-equi --topics /zed/left/image_raw /zed/right/image_raw --bag camera-imu-data.bag --target aprilgrid_6x6.yaml
+```
+Arguments:<br/>
+-models: Most typically used stereo cameras are of pinhole camera model. For more detail on the supported models, refer to this [link](https://github.com/ethz-asl/kalibr/wiki/supported-models).<br/>
+-topics: Name of the recodered camera topics  <br/>
+-bag: ROS bag containing the image and IMU data <br/>
+-target: yaml configuration file for the used target <br/>
+
+This command will generate a new yaml file with camera intrinsic calibration parameters. Above command can be skipped if the camera calibration yaml is already available. Now run below imu-camera calibration script.
```

Comment:
HTML tags should be avoided and aren't necessary here. 

However, I think you'd prefer adding a space after each `-`. For instance, `- models` not `-models` to create bullet points.

If you're having trouble with lines merging, then use two empty lines to distinguish separate paragraphs.